src/main/java/com/nutomic/syncthingandroid/activities/LogActivity.java

```diff
@@ -0,0 +1,70 @@
+package com.nutomic.syncthingandroid.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.widget.EditText;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.util.Logger;
+
+/**
+ * Shows the log information from Syncthing.
+ */
+public class LogActivity extends SyncthingActivity {
+
+    public static Logger logger = new Logger(2000);
+
+    private EditText mLog;
+    private boolean syncthingLog = true;
+
+    /**
+     * Initialize Log.
+     */
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.log_activity);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        mLog = (EditText) findViewById(R.id.log);
+        mLog.setText(logger.getLog());
+    }
+
```

Comment:
Needs to implement `onSaveInstanceState()` and save/restore android/native log state.